CAP/gap/OppositeCategory.gi

```diff
@@ -65,6 +65,7 @@ BindGlobal( "CAP_INTERNAL_FIND_OPPOSITE_PROPERTY_PAIRS",
         else
             current_entry := [ current_rec.installation_name,
                                CAP_INTERNAL_METHOD_NAME_RECORD.( current_rec.dual_operation ).installation_name ];
+            current_entry := [ Concatenation( current_entry[ 1 ], " vs ", current_entry[ 1 ] ), current_entry ];
```

Comment:
Do you mean `current_entry[ 1 ], " vs ", current_entry[ 2 ] `?

Reply:
Could you explain the fix?
Does every ToDoList entry need a description?

Reply:
>  Do you mean current_entry[ 1 ], " vs ", current_entry[ 2 ]?

Yes

> Could you explain the fix?
> Does every ToDoList entry need a description?

No, not in general, but in this case it requires one. The reason is the specification of the entries of the list passed as third argument of `ToDoListEntryToMaintainFollowingAttributes`:

If it is only one string, it is interpreted as attribute name for both source and target of the entry. If it is a list of two strings, the first string is interpreted as description of the entry, the second as attribute name for both source and target. The only way to add two different attributes right now is to add a description.

I think with a little refactoring of the logic of `ToDoListEntryToMaintainFollowingAttributes` we could make the entry a list of a list containing two strings. But for now, this works. Maybe I should post an issue to ToolsForHomalg?